test-plan.md

```diff
@@ -93,9 +93,11 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 * In a project that uses globbing (.NET Core), use the VS Code file explorer to add a new file next to the csproj. Intellisense/sighelp/etc should be available in the new file
 * Add a new file and reference a type in it from a different file. Deleting from disk the file containing the referenced type  should produce error messages
 
-#### ASP.NET Core Razor
-The Razor experience is available when you open a .cshtml file in a valid OmniSharp project. To setup a test project to verify on you can do:  
-1. `dotnet new razor`
+#### ASP.NET Core Razor Web App
```

Comment:
A "Razor Web App" isn't a term we use anywhere else.  Let's just name this section "Razor"